src/components/screens/sprint-view-screen.tsx

```diff
@@ -104,44 +105,9 @@ const SprintViewScreen = () => {
   }
 
   /**
-   * Get total time required to complete the project 
-   */
-  const getTotalTimeEntries = (allocation: Allocations) => {
-    if (timeEntries.length!==0) {
-      return timeEntries[allocations.indexOf(allocation)];
-    }
-    return 0;
-  }
-
-  /**
-   * Calculate the remaining time of project completion
-   */
-  const timeLeft = (allocation: Allocations) => {
-    return totalAllocations(allocation) - (getTotalTimeEntries(allocation) || 0)
-  }
-
-  /**
-   * Get project name
-   */
-  const getProjectName = (allocation: Allocations) => {
-    if (projects.length !== 0) {
-      return projects[allocations.indexOf(allocation)]?.name;
-    }
-    return "";
-  }
-
-  /**
-   * Get project color
-   */
-  const getProjectColor = (allocation: Allocations) => {
-    if (projects.length !== 0) {
-      return projects[allocations.indexOf(allocation)]?.color;
-    }
-    return "";
-  }
-
```

Comment:
You moved these in to sprint-project-columns but perhaps they would be better place in a sprint utils file?